.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
Many maintainers dislike the addition of editor-specific ignores. Please use a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) to ignore your editor's project files.

Reply:
Ok, I fixed it. But this project is under maintenance yet?